charts/eks-anywhere-packages/templates/deployment.yaml

```diff
@@ -146,6 +148,10 @@ spec:
           defaultMode: 420
           secretName: ecr-token
           optional: true
+      - name: aws-secret
```

Comment:
what do you think about `aws-config-secret`? or is `aws-secret` the convention elsewhere?

checking my understanding: we can't use the existing ecr-token secret here because we don't have control over its renewal lifecycle?

Reply:
"aws-secret" contains a field called "config", calling it "aws-config-secret" may imply that the volume is the "config" of "aws-secret".

ecr-token stores "username:password", it's not refreshed anymore perhaps because we have migrated to use credential-provider.